capture_service/android_application.h

```diff
@@ -61,8 +70,13 @@ class AndroidApplication
     ApplicationType m_type;
     std::string     m_main_activity;
     std::string     m_command_args;
+    std::string     m_device_architecture;
```

Comment:
could you put some comments on what are the available device architecture options?